Lilypad-Tech/lilypad
# refactor: remove doppler depedency from local development

### Review Type Requested (choose one):

### Summary
This PR looks to remove Doppler as a dependency during local development so that anybody can run the app locally

### Task/Issue reference

Closes: #278 

### Details (optional)
Add any additional details that might help Code Reviewers digest this PR

### How to test this code? (optional)

1. Pull Code
2. Run app through individual stack commands (see local dev guide)
3. Do a run of cowsay (onchain and off-chain) and verify app functions as expected
4. Run Integration test locally and verify test passed
5. Stop the app
6. Now run the app using the docker stack commands (see local dev guide)
7. Repeat steps 3 to 4

### Anything else? (optional)
